message.js

```diff
@@ -12,19 +12,21 @@
 'use strict';
 
 module.exports = function(
+  pull_requester: string,
   reviewers: Array<string>,
   mentionSentenceBuilder: (reviewers: Array<string>) => string,
-  defaultMessageGenerator: (reviewers: Array<string>) => string
+  defaultMessageGenerator: (pull_requester: string, reviewers: Array<string>) => string
```

Comment:
Can you swap the two (make pullRequester second) so that it is not a breaking change and existng code keeps working